identities/omni1979/info.json

```diff
@@ -1,7 +1,7 @@
 {
-  "description": "“It’s a dangerous business, going out your door. You step onto the road, and if you don’t keep your feet, there’s no knowing where you might be swept off.\"",
+  "description": "\"The Shire at this time had hardly any ‘government’. Families for the most part managed their own affairs.\"",
```

Comment:
remove quote escaping ?

Reply:
this is directly taken from here: https://github.com/omni1979